.env

```diff
@@ -1,2 +1,2 @@
-PORT=8083
+PORT=8014
```

Comment:
PORT is set here but is hardcoded in Dockerfile and yaml files. Can it be improved?

Reply:
I think @Mehrnaz-Charkhchi  can comment on this. But my interpretation is .env is used for local development if one  wants to run it at different port then this is the place where it should be modified.

Reply:
Wouldn't you normally commit an `.env.sample`, from which the developer then creates a git-ignored `.env` file with appropriate modifications for local development? Such as is done in the [genome browser repository](https://github.com/Ensembl/ensembl-dauphin-style-compiler/tree/master/configurations/dev)?

_(Just curious. I don't have a strong opinion on the matter.)_